vapi/simulator/simulator.go

```diff
@@ -333,6 +332,16 @@ func OK(w http.ResponseWriter, val ...interface{}) {
 	StatusOK(w, s)
 }
 
+// OKWithNoContent responds with http.StatusNoContent.
+func OKWithNoContent(w http.ResponseWriter) {
```

Comment:
nit: why not directly call `StatusNoContent`? Or rename `StatusNoContent` to `OKStatusNoContent` and use this instead?

Reply:
sorry for not being clear perhaps: what I meant was why do we have this function if the only thing it does is calling another function?

Reply:
I wanted to follow the existing format:

currently, to indicate 200, we call OK() which calls StatusOk()

similarly, we added

To indicate 204, call OKWithNoContent() which will be calling StatusNoContent()